src/main.cpp

```diff
@@ -291,6 +358,85 @@ int main(int argc, char **argv){
         if (visualize)
             visualization::visualize(world, robots, collision, config.show_full_map.value(), bbox, robot_radius);
 
+        visualization_msgs::Marker doors, moving_objects, undefined_objects;
+        doors.header.frame_id = "origin";
+        doors.header.stamp = ros::Time::now();
+        doors.ns = "geometry";
+        doors.action = visualization_msgs::Marker::ADD;
```

Comment:
Although `add` and `modify` are the same in the message definition. Better to use `modify` here to indicate newer messages overrule older messages instead of appending older messages.